# Integration-Tool-for-scRNA-seq-Data-and-Spatial-Transcriptomics-Sequencing-Data

## Introduction

In scRNA-seq, tissue dissociation is a crucial step that results in the loss of spatial information pertaining to cells in the original tissue. Consequently, integrating spatial transcriptomics data with scRNA-seq data for combined analysis holds the potential to significantly enhance our comprehension of biological mechanisms in both spatial and temporal dimensions.

Integration methods of scRNA-seq and spatial transcriptomics can be broadly classified into two main categories: Deconvolution and Mapping. Deconvolution methods involve the construction of mathematical or statistical inference models, wherein scRNA-seq data serves as the background data. These methods integrate the spatial transcriptomics sequencing data with the known cell types or expression patterns of marker genes derived from scRNA-seq data, enabling the inference of cell types for each spot. Conversely, mapping aims to establish correspondence between scRNA-seq data and spatial transcriptomics sequencing data, aligning and mapping them within the spatial domain. This approach enables spatial visualization and analysis of cell types or gene expression patterns. In contrast to the deconvolution approach, mapping typically doesn't necessitate the prior development of elaborate cell subtype models. Mapping is generally more flexible and better suited for situations demanding greater precision in cell source identification. Spatial transcriptomics sequencing data typically involve two sequencing methods: HPRI and Spatial barcoding technology. HPRI technology provides greater sequencing depth, resulting in higher RNA molecule abundance at each genomic location, while Spatial barcoding offers slightly higher resolution, allowing for better distinction and spatial precision among different locations.

In this article, further categorization is based on the use of Highly Variable Genes (HVGs). Integration of single-cell RNA sequencing data and spatial transcriptomics data typically involves the identification of Highly Variable Genes (HVGs), which are genes exhibiting significant expression differences within cell populations. These differentially expressed genes may play crucial regulatory roles in terms of cell types, states, and other aspects. Classifying integration techniques into two subcategories based on the use of HVGs, as opposed to previous methods, better highlights the selection of integration techniques for achieving either a global or local perspective. This distinction is particularly valuable in scenarios involving large datasets and requiring significant computational resources for HVG computation, as depicted in the "Deconvolution" and "Mapping" sections below.

![image](https://github.com/ChaoruiYan019/Integration-Tool-for-scRNA-seq-Data-and-Spatial-Transcriptomics-Sequencing-Data/blob/main/iamges/Deconvolution.JPG)

![image](https://github.com/ChaoruiYan019/Integration-Tool-for-scRNA-seq-Data-and-Spatial-Transcriptomics-Sequencing-Data/blob/main/iamges/Mappng.JPG)
